src/styles/_variables.scss

```diff
@@ -71,18 +71,23 @@ $textarea-default-max-height: 200px;
 $compact-table-row-height: $input-default-height;
 $pagination-item-height: 18px;
 $grid-row-min-height: 45px;
+$sidebar-default-height: 100%;
 
 /* Width */
 $button-container-max-width: 500px;
 $button-max-width: 200px;
 $button-min-width: 100px;
 $input-compact-width: 70px;
+$sidebar-default-width: 280px;
+$sidebar-max-width: 70%;
+$sidebar-logo-default-width: 40%;
```

Comment:
we generally use in variables reusable styles. If you see that some width will only be used once, define in not in the general variables file, but in the component specific scss file only.

please, review all your codes in this aspect.

Reply:
moved to app scss file